apps/notifications-test-util/src/index.ts

```diff
@@ -19,32 +19,40 @@ async function main(): Promise<void> {
 
   console.log(`Sending notification to: ${token}.`)
 
-  //
+  // //
+  // await getMessaging().send({
+  //   token,
+  //   data: {
+  //     type: 'TEST',
+  //   },
+  //   android: {
+  //     priority: 'high',
+  //   },
+  //   apns: {
+  //     payload: {
+  //       aps: {
+  //         'content-available': true,
+  //       },
+  //     },
+  //     headers: {
+  //       'apns-push-type': 'background',
+  //       'apns-priority': '5',
+  //       'apns-topic': 'it.vexl.next', // your app bundle identifier
+  //     },
+  //   },
+  // })
```

Comment:
do we really need this commented section?

Reply:
It's nice to have examples how to send notifications differently. This package is meant as a playground for sending notifications. 

I believe we will find this code useful once we try to debug why notifications are not being received reliably.